charts/graph-network-indexer/values.yaml

```diff
@@ -35,71 +38,113 @@ grafana:
-  config:
-    # -- (required) URL for your graph-node admin API endpoint
-    graph-node-admin-endpoint: null
-    # -- (required) Public HTTPS URL of your indexer-service query endpoint
-    public-indexer-url: null
-    # -- (required) A command separated list of graph-node Node IDs to assign subgraphs to
-    index-node-ids: null
-    # -- (required) The gateway collect-receipts endpoint for getting vouchers
-    collect-receipts-endpoint: null
-    # -- (required) Base58 deployment hash (Qm...) for the Graph Network Subgraph
-    network-subgraph-deployment: null
-    # -- (required) Contract address of ERC20 used for DAI variable in cost models
-    dai-contract: null
-    # -- (optional) Query endpoint for syncing status of EBO and its contract state.
-    epoch-subgraph-endpoint: null
-
-  env: {}
+  config: {}
+      # # -- (required) URL for your graph-node admin API endpoint
+      # graph-node-admin-endpoint: null
+      # # -- (required) Public HTTPS URL of your indexer-service query endpoint
+      # public-indexer-url: null
+      # # -- (required) A command separated list of graph-node Node IDs to assign subgraphs to
+      # index-node-ids: null
+      # # -- (required) The gateway collect-receipts endpoint for getting vouchers
+      # collect-receipts-endpoint: null
+      # # -- (required) Base58 deployment hash (Qm...) for the Graph Network Subgraph
+      # network-subgraph-deployment: null
+      # # -- (required) Contract address of ERC20 used for DAI variable in cost models
+      # dai-contract: null
+      # # -- (optional) Query endpoint for syncing status of EBO and its contract state.
+      # epoch-subgraph-endpoint: null
 
   secretEnv: {}
+    # -- We recommend that you set the following environment variables in the indexer-agent secret
+    # INDEXER_AGENT_MNEMONIC:
+    #   secretName: # your secret containing mnemonic
+    #   key: # your key in the secret
+    # INDEXER_AGENT_POSTGRES_USERNAME:
+    #   secretName: # your secret containing postgres username
+    #   key: # your key in the secret
+    # INDEXER_AGENT_POSTGRES_PASSWORD:
+    #   secretName: # your secret containing postgres password
+    #   key: # your key in the secret
+
+  env: {}
 
   # -- Additional CLI arguments to pass to `indexer-agent`
-  extraArgs: []
+  extraArgs: 
+    # -- (required) URL for a blockchain node that has the Graph Protocol contracts (e.g. Ethereum Mainnet, Goerli)
+    - --ethereum=null
+    # -- Name of the network that you have specified a node URL for in `ethereum`
+    - --ethereum-network="mainnet"
+    # -- (required) URL for your graph node query endpoint (probably a load balancer address)
+    - --graph-node-admin-endpoint=null
+    # -- (not recommended) Specify a plain text mnemonic for your operator account. Instead, we recommend using a Kubernetes Secret and secretEnv to mount the value as an environment variable.
+    - --mnemonic=null
```

Comment:
why are we going with flags for setting these parameters here? instead of config keys and then settings these flags as a function of those keys :thinking: